### PR TITLE
Update target user and user stories in DG

### LINKED ISSUE #31: Update DG - Target user profile, value proposition, and user stories

### PR DESCRIPTION
**Pull request by Javiery3889**
- Update target user to financial advisors
- Update value proposition to align with group's proposed value proposition
- Update user stories table

Closes #31

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -262,29 +262,50 @@ _{Explain here how the data archiving feature will be implemented}_
...
+| `* *`    | Financial Advisor | filter contact list by meeting details                                             | find out who I'm meeting                                                        |
+| `* *`    | Financial Advisor | modify a scheduled meeting's details                                               | keep up-to-date with the meeting's details                                      |
+| `* *`    | Financial Advisor | shift clients into different groups                                                | reorganize in the event of changes                                              |
+| `* *`    | Financial Advisor | be able to add simple notes to my client contact information                       | know their financial goals to prepare me for my next meeting with them.         |
+| `*`      | New user          | import contact information in bulk to FINDvisor                                    | save time and ensure no client is overlooked.                                   |
```
Comment (KohGuanZeh) [inline docs/DeveloperGuide.md:296]: I think we should remove the punctuation at the end to keep it consistent? Just a nit.

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -262,29 +262,50 @@ _{Explain here how the data archiving feature will be implemented}_
...
+| `* * *`  | New user          | know how to operate the basic functionalities of FINDvisor within the app         |                                                                                 |
+| `* * *`  | Financial Advisor | add contacts of my clients                                                         | keep a record of my clients' contact information.                                |
+| `* * *`  | Financial Advisor | find contacts of my clients                                                        | find information on a specific client.                                          |
+| `* * *`  | Financial Advisor | update client's contact information                                                | not need to delete and create new contact information.                           |
+| `* * *`  | Financial Advisor | remove contact information                                                         | reduce clutter in contact list with clients I have severed ties with.           |
```
Comment (KohGuanZeh) [inline docs/DeveloperGuide.md:286]: I think we should remove the punctuation at the end to keep it consistent? Just a nit.

**Review by KohGuanZeh — COMMENTED** [review]
LGTM. Just some small nits.

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -262,29 +262,50 @@ _{Explain here how the data archiving feature will be implemented}_
 
 **Target user profile**:
 
-* has a need to manage a significant number of contacts
-* prefer desktop apps over other types
+* financial advisors
+* has a need of scheduling meetings with a significant number of clients.
```
Comment (KohGuanZeh) [inline docs/DeveloperGuide.md:266]: I think the punctuation should be removed for consistency? Just a small nit.

**Review by KohGuanZeh — APPROVED** [review]
LGTM. Only a small nit that I missed from last review.

File: docs/DeveloperGuide.md (hunk 3 above)
Comment (KohGuanZeh) [inline docs/DeveloperGuide.md:266]:
```suggestion
* has a need of scheduling meetings with a significant number of clients
```
I think punctuation should be removed for consistency?

**Review by KohGuanZeh — APPROVED** [review]

**Review by KohGuanZeh — APPROVED** [review]

**Review by Dethada — APPROVED** [review]

**Pull request merged**